### ISSUE TITLE
Heart attacks need more feedback

### ISSUE THREAD

**Issue by PKPenguin321** [tgstation/-tg-station#10276]
Heart attacks, as far as I know, are only caused in normal play by getting shocked with an emagged defibrillator on harm intent. To fix them, you must be shocked by a defib on help intent. They cause you to take extreme amount of oxygen damage, enough to put you in critical condition very quickly.

I was doing some testing with them in a private test server with them in a private server, and it was very hard to tell what was intentional and what was not when it came to curing heart attacks. The defib would tell me the patient's heart was beating again, but they would continue to take extreme oxygen damage for some time (even when not in crit), anyways. I came to report this as a bug, but after a very long while (a few minutes at least) of keeping the patient barely alive with CPR and epinephrine, it seemed to stop with no real indication.

tl;dr: There needs to be more feedback on whether somebody is having a heart attack or not (maybe make it show in medical analyzers), and there needs to be feedback on whether or not it is cured/if it is in the process of being cured/how to cure it.

**Comment by Boredone** [comment]
Heart Attacks can also be caused by one of the poisons traitors can order. Otherwise I agree.

**Comment by CosmicScientist** [comment]
Did you check which chems were in their system or if their blood level was high?

**Comment by PKPenguin321** [comment]
Blood level was untouched as no brute damage was done

At times they had epinephrine and minor amounts of tricord in their blood so they wouldn't die immediately

**Comment by CosmicScientist** [comment]
That does sound weird.

**Comment by phil235** [comment]
The health analyzer shows whether your patient has a heart attack and tells you to apply a defibrillator. Also every time someone gets their heart_attack var set to 1 it is accompanied by a very explicit visible_message(). The only thing missing would be a message when the heart attack is cured on its own.
